src/coreclr/vm/interpexec.cpp

```diff
@@ -321,21 +321,28 @@ static void InterpBreakpoint()
 }
 #endif
 
+#ifdef FEATURE_REUSE_INTERPRETER_STACK_FOR_NORMAL_FUNCLETS
+#define LOCAL_VAR_ADDR(offset,type) ((type*)(stack + (offset)))
```

Comment:
we do stack arithmetic in a bunch of other places, and I don't see them all patched here. i.e. `stack + callArgsOffset` when doing calls. Is there a comment somewhere in this PR explaining the rule(s) for when that is safe or isn't safe? Should we ensure we never do raw stack arithmetic in this file (outside of these macros/helpers) so that everything will be consistent?

Reply:
stack + callArgsOffset is the only one which shouldn't be patched other than frame setup. And even there it COULD be patched, but it had better never be negative, since a negative value would just result in stack overwriting. (since the callArgsOffset is used to establish the new stack frame in the called function.).